packages/form-state-manager/src/utils/manager-api.ts

```diff
@@ -5,6 +5,7 @@ import CreateManagerApi, { ManagerState, ManagerApi, AsyncWatcher, AsyncWatcherR
 import AnyObject from '../types/any-object';
 import FieldConfig from '../types/field-config';
 import { formLevelValidator } from './validate';
+import { values } from 'lodash';
```

Comment:
Maybe I am blind, but I can't see this used anywhere. If it is, please use direct import `lodash/values`. Or consider using `Object.values` instead.